### PR TITLE
Adds kill assist history

### PR DESCRIPTION
**Pull request by Anthony-De**
Kill-assist needs to be resolved before this is added.
killer-name (assisted) ⚔️ victim-name
changes limit to 33%

### CODE REVIEW

**Comment by moreavy** [comment]
@Anthony-De No files changed?

**Comment by Anthony-De** [comment]
I'm still working on it @moreavy

**Comment by Anthony-De** [comment]
It is now ready for testing. @moreavy

File: mods/ctf/ctf_events/init.lua [hunk 1]
```diff
@@ -120,7 +122,82 @@ function ctf_events.update_all()
 	end
 end
 
+local good_weapons = {
```
Comment (moreavy) [inline mods/ctf/ctf_events/init.lua:125]: It seems the knight's sword is missing here.

Comment (Anthony-De) [reply mods/ctf/ctf_events/init.lua:125]: I added it.

**Pull request closed without merging**